rdr_service/resource/calculators/participant_ubr.py

```diff
@@ -120,13 +120,13 @@ def ubr_ethnicity(answers):
         :return: UBRValueEnum
         """
         if answers is None:
-            return UBRValueEnum.NullSkip
+            return UBRValueEnum.NotAnswer_Skip
```

Comment:
To make sure I understand, a "missing" answer is supposed to be treated the same as a PMI_PreferNotToAnswer or PMI_Skip answer code?

Reply:
The details of that are to be worked out in the AST meeting, I wanted to change the enum name to something that represents the current state of the code.

Reply:
Ok, but the "current state of the code" is also treating null/missing answer codes the same as the legit PMI_ codes that can be present?  Not asking for an enum change, just wanted to be sure there wasn't supposed to be any distinct logic for null/missing answers in this PR.  Sounds like that may change pending outcome of AST meeting?